dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/bundle/persister/EnrollmentPersister.java

```diff
@@ -158,4 +162,14 @@ protected void runPreCreateHooks( TrackerBundle bundle )
         bundle.getEnrollments()
             .forEach( o -> bundleHooks.forEach( hook -> hook.preCreate( Enrollment.class, o, bundle ) ) );
     }
+
+    @Override
+    protected void persistOwnership( TrackerPreheat preheat, ProgramInstance entity )
+    {
+        if ( isNew( preheat, entity.getUid() ) )
```

Comment:
Could we remove the isNew check if we change parameter overwriteIfExists to false?

Reply:
Good point. 2 reasons I can think of.
1. We would want to overwrite the ownership in case of "every" new enrollment. If we change to overwrite-false, we actually leave a risk of leaving the ownership record unchanged for a "second" new enrollment created. 
2. The isNew check seems to be less expensive as it does not involve a db call? But the overwrite check will still fetch the record. In the end both will display the same behaviour.

Reply:
It makes sense